app/components/curriculum-inventory-sequence-block-session-list.js

```diff
@@ -4,28 +4,28 @@ import { computed } from '@ember/object';
 import { task } from 'ember-concurrency';
 
 export default Component.extend({
-  linkedSessions: null,
+  sessions: null,
   linkableSessionsBuffer: null,
   classNames: ['curriculum-inventory-sequence-block-session-list'],
   tagName: 'section',
   sortBy: 'title',
 
   init() {
     this._super(...arguments);
-    this.set('linkedSessions', []);
+    this.set('sessions', []);
```

Comment:
Is this needed?  It's going to just end up being the value passed in right?

Reply:
yes, it's expected to be passed in. i'll see what happens when i yank it.

Reply:
this checks out w/o initializing. i removed that line.